java/parser/src/main/proto/Parser.proto

```diff
@@ -25,6 +25,12 @@ message ParserRequest {
   string sql_program = 2;
 }
 
+// InputOutputTables represents input and output tables a statement manipulates.
+message InputOutputTables {
```

Comment:
I slightly suggest shortening the name to `IOTables`, like `<iostream>` in CPP

Reply:
The name `IOTables` causes:

```
[WARN] /Users/wuyi/go/src/sqlflow.org/sqlflow/java/parse-interface/src/main/java/org/sqlflow/parser/parse/ParseResult.java:15:30: Member name 'IOTables' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName]
[WARN] /Users/wuyi/go/src/sqlflow.org/sqlflow/java/parse-interface/src/main/java/org/sqlflow/parser/parse/ParseResult.java:15:30: 名称 'IOTables' 中不能出现超过 '2' 个连续大写字母。 [AbbreviationAsWordInName]
```